util/src/windows/scoped_handle.cc

```diff
@@ -0,0 +1,17 @@
+#include <leatherman/util/windows/scoped_handle.hpp>
+#include <windows.h>
+
+using namespace std;
+
+namespace leatherman { namespace util { namespace windows {
+
+    scoped_handle::scoped_handle(HANDLE h) :
+        scoped_resource(h, CloseHandle)
```

Comment:
`CloseHandle` is smart enough, but tools like AppVerifier might see it as a bug (it'll certainly warn you if you call it with an invalid handle, but I'm not sure if it considers `INVALID_HANDLE_VALUE` to itself be invalid; can't hurt to guard against it though).